src/components/RadioGroup/RadioGroup.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { States } from 'storybook-states';
+import { RadioButton } from '../RadioButton';
+import { RadioGroup } from './RadioGroup';
+import { Text } from '../Text';
+import { Stack } from '../Stack';
+
+export default { title: 'Components / RadioGroup' };
+
+export const Default = () => (
+  <States>
```

Comment:
I think `States` is unnecessary here.

Reply:
it displays that wrapper, makes things look consistent

Reply:
this:
<img width="337" alt="Screenshot 2020-10-23 at 13 30 14" src="https://user-images.githubusercontent.com/43396056/96993385-e8e88500-1533-11eb-8b12-e6055366cf1e.png">